apps/playground/src/app/modules/components/upload-module/upload-module.component.html

```diff
@@ -0,0 +1,188 @@
+<section class="module-description">
+  <h2>DsfrUploadComponent</h2>
+  <p>
+    The <code>DsfrUploadComponent</code> purpose is to represent an input file form
+    field
+  </p>
+  <p>
+    It is based on the
+    <a
+      target="_blank"
+      hreflang="fr"
+      href="https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/ajout-de-fichier/">
+      Upload
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    component in the
+    <a
+      target="_blank"
+      hreflang="fr"
+      href="https://www.systeme-de-design.gouv.fr/">
+      DSFR
+      <mat-icon>open_in_new</mat-icon>
+    </a>
+    .
+  </p>
+  <p>It has several <code>&#64;Input()</code>  and <code>Output()</code> properties:</p>
+  <dl class="input-properties">
+    <play-input-property-definition
+      name="id"
+      [required]="true"
+      type="It takes a string">
+      It is required and given to the <code>id</code> attribute of the
+      underlying <code>input</code> tag. It is also
+      used to generate the <code>aria-describedby</code> attribute and IDs for
+      the error message if needed.
+    </play-input-property-definition>
+    <play-input-property-definition
```

Comment:
```html
<play-input-property-definition
      name="label"
      default="'Ajouter un fichier' or 'Ajouter des fichiers' if multiple is true"
      type="It takes a string">
      The label that will be displayed on top of the form field
    </play-input-property-definition>
    <play-input-property-definition
      name="hint"
      [required]="true"
      type="It takes a string">
      <p>
        The text that will be displayed beneath the label. It MUST describe all
        the expected restrictions about formats and sizes.
      </p>
      <p>
        ⚠️Be careful ⚠️, these restrictions are not implemented within this
        Component. It is still your responsibility to check them on the server
        level
      </p>
    </play-input-property-definition>
    <play-input-property-definition
      name="accept"
      type="string">
      <p>
        A list of comma-separated extensions so that the dialog box opened by
        clicking on this Component will only let you choose the expected files.
        For instance : ".jpg, .jpeg, .png"
      </p>
      <p>
        ⚠️Be careful ⚠️, it is still possible to manually change the extension
        in the name of the file and make it available for choosing. This
        attribute should NOT be seen as a security check but rather like a
        user-experience bonus
      </p>
    </play-input-property-definition>
    <play-input-property-definition
      name="multiple"
      default="false"
      type="boolean">
      It will allow a user to upload more than 1 file
    </play-input-property-definition>
```